### PR TITLE
Add `Label::halign`

### PR DESCRIPTION
**Pull request by rustbasic**
Function to specify `Align` to `Label`

There are cases where you want to display `Label` on the left or right regardless of other `Layout` specifications.

Before : Note the part showing the rust source code.

After : Note the part showing the rust source code.

### CODE REVIEW

**Comment by bircni** [comment]
maybe change the function name to text_align as this only aligns the text inside the label not the label itself

**Comment by rustbasic** [comment]
> maybe change the function name to text_align as this only aligns the text inside the label not the label itself

Yes, that would be great too. Let's wait for emilk's review.

File: crates/egui/src/widgets/label.rs [hunk 1]
```diff
@@ -76,6 +78,13 @@ impl Label {
         self
     }
 
+    /// Set [`Self::align`] to [`Align`].
```
Comment (emilk) [inline crates/egui/src/widgets/label.rs:81]: This doesn't explain anything

File: crates/egui/src/widgets/label.rs [hunk 2]
```diff
@@ -76,6 +78,13 @@ impl Label {
...
+    #[inline]
+    pub fn align(mut self, align: Align) -> Self {
+        self.align = Some(align);
```
Comment (emilk) [inline crates/egui/src/widgets/label.rs:84]: `halign` is probably a better name

**Review by emilk — CHANGES_REQUESTED** [review]

File: crates/egui/src/widgets/label.rs [hunk 3]
```diff
@@ -76,6 +78,15 @@ impl Label {
         self
     }
 
+    /// Sets the horizontal alignment of the Label to the given `Align` value.
+    ///
+    /// Set [`Self::halign`] to [`Align`].
```
Comment (emilk) [inline crates/egui/src/widgets/label.rs:83]:
```suggestion
```

**Pull request merged**